src/wallOverlap.cpp

```diff
@@ -128,26 +129,14 @@ int64_t WallOverlapComputation::handlePotentialOverlap(const ListPolyIt from_it,
 
 int64_t WallOverlapComputation::getApproxOverlapArea(const Point from, const Point to, const int64_t to_dist, const Point other_from, const Point other_to, const int64_t from_dist)
 {
-    const int64_t overlap_width_2 = line_width * 2 - from_dist - to_dist; //Twice the width of the overlap area, perpendicular to the lines.
-
     // check whether the line segment overlaps with the point if one of the line segments is just a point
     if (from == to)
     {
-        if (LinearAlg2D::pointIsProjectedBeyondLine(from, other_from, other_to) != 0)
-        {
-            return 0;
-        }
-        const int64_t overlap_length_2 = vSize(other_to - other_from); //Twice the length of the overlap area, alongside the lines.
-        return overlap_length_2 * overlap_width_2 / 4; //Area = width * height.
+        return 0;
     }
     if (other_from == other_to)
     {
-        if (LinearAlg2D::pointIsProjectedBeyondLine(other_from, from, to) != 0)
-        {
-            return 0;
-        }
-        const int64_t overlap_length_2 = vSize(from - to); //Twice the length of the overlap area, alongside the lines.
-        return overlap_length_2 * overlap_width_2 / 4; //Area = width * height.
```

Comment:
Yeah, well, I didn't like the end results that the above code produced. It's a while now since I changed it but if my memory serves me, it was too aggressive. I will go back and check again the difference leaving that out makes.